benchexec/tablegenerator/columns.py

```diff
@@ -18,12 +18,10 @@
 
 __all__ = ["Column", "ColumnType", "ColumnMeasureType"]
 
-# This sets the rounding mode for all Decimal operations in the process.
-# It is actually used only as default context for new contexts, but because we set this
-# at import time and before any threads are started, it should work according to its
-# documentation. We double check with the context of the current thread.
-decimal.DefaultContext.rounding = decimal.ROUND_HALF_UP
-assert decimal.getcontext().rounding == decimal.ROUND_HALF_UP
+# Important: It's important to make sure on *all* entry points / methods which perform arithmetics that the correct
+#            rounding / context is used.
+local_context = decimal.getcontext()
```

Comment:
This feels more like a module-level constant, so, I would use CONSTANT_CASE. And name would be better if it "decimal" ("rounding" would also be good at first thought, but the context defines more than that). "local context" could be anything. Just `DECIMAL_CONTEXT` would already be better.

Furthermore, this is a weird strategy: Setting the rounding mode on the current thread's context, but then *also* using `with decimal.localcontext`. If we already bite the bullet of having to use `localcontext`, then we do not need to and should not change the current thread's context and affect other code. So we should create a new context.